src/dbt/powerschool/models/base/base_powerschool__final_grades.sql

```diff
@@ -80,6 +80,8 @@ with
 
             sgs.grade_points as sg_grade_points,
 
+            fg.gradelastupdate,
```

Comment:
@GabyRangelB just want to make sure this is the field you want. does it behave the same as whenmodified/transaction_date?